### DISCUSSION TITLE
Mobile to Yetiforce

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#15476] [How to]
As here is working module ready to connct easily to popular SMS gateways, I have a question related to mobile devices.
What about connceting yeti directly to Android/iOS phones and sending TXT meesages this way? Does anyone was considering/developing a such?

**Comment by apcloic** [comment]
If you want to turn an android smartphone to an sms gateway, you can use something like this :
http://smssync.ushahidi.com/
I've been doing some test with it a few months ago and it was working properly but I didn't go further. 
It's also Open source and very easy to integrate into YF.

**Reply by mariuszkrzaczkowski** [reply]
interesting idea

**Comment by Marshal4582** [comment]
Yp, this is interesting potential enchancement.
